mpa/cls/stage.py

```diff
@@ -196,12 +199,14 @@ def configure_task(cfg, training, model_meta=None, **kwargs):
                 # model configuration update
                 cfg.model.head.num_classes = len(dst_classes)
                 gamma = 2 if cfg['task_adapt'].get('efficient_mode', False) else 3
-                cfg.model.head.loss = ConfigDict(
-                    type='SoftmaxFocalLoss',
-                    loss_weight=1.0,
-                    gamma=gamma,
-                    reduction='none',
-                )
+
+                if 'Multi' not in cfg.model.head.type:
```

Comment:
As in the previous comment, we can use (head.type ->) cfg.task as the standard task type.

Reply:
Same as above comment.